ouroboros-consensus/src/unstable-mock-block/Ouroboros/Consensus/Mock/Protocol/Praos.hs

```diff
@@ -618,6 +627,40 @@ instance PraosCrypto PraosMockCrypto where
   type PraosVRF  PraosMockCrypto = MockVRF
   type PraosHash PraosMockCrypto = SHA256
 
+{-------------------------------------------------------------------------------
+  Orphan instances to make Ed448 look like a proper mlocked DSIGN
+-------------------------------------------------------------------------------}
+
+instance DSIGNMAlgorithm Ed448DSIGN where
+  data SignKeyDSIGNM Ed448DSIGN = SignKeyDSIGNMEd448 (SignKeyDSIGN Ed448DSIGN)
+    deriving (Generic)
+
+  deriveVerKeyDSIGNM (SignKeyDSIGNMEd448 sk) =
+    return $ deriveVerKeyDSIGN sk
+
+  signDSIGNM context signable (SignKeyDSIGNMEd448 sk) =
+    return $ signDSIGN context signable sk
+
+  genKeyDSIGNMWith _ mlockedSeed = do
+    seed <- mlockedSeedUseAsCPtr mlockedSeed $ \seedPtr -> do
+      mkSeedFromBytes <$> packByteStringCStringLen (castPtr seedPtr, fromIntegral $ seedSizeDSIGN (Proxy :: Proxy Ed448DSIGN))
+    return . SignKeyDSIGNMEd448 $ genKeyDSIGN seed
+
+  cloneKeyDSIGNMWith _ sk = return sk
+
+  getSeedDSIGNMWith _ _ _ =
+    error "getSeed not implemented for Ed448"
+
+  forgetSignKeyDSIGNMWith _ _ =
+    -- This is fake mlocking, so just don't forget anything
+    return ()
```

Comment:
Are we using this still? I see above you changed to ED25519DSIGN

Reply:
Indeed this is no longer needed.